docs/windows.md

```diff
@@ -0,0 +1,60 @@
+# Experimental Windows support
+
+As from `v0.13.0-rc2`, Buildkit now has exprimental support on Windows. Both `buildctl.exe` and `buildkitd.exe` binaries are being released for testing purposes.
+
+We will apprecate any feedback by [opening an issue here](https://github.com/moby/buildkit/issues/new), as we stabilize the product, especially `buildkitd.exe`.
+
+## Quick start guide
+
+### Platform requirements
+
+- Architecture: `amd64`, `arm64` (binaries available but not officially tested yet).
+- Supported OS: Windows Server 2022, Windows 11.
+- Base images: `ServerCore:ltsc2022`, `NanoServer:ltsc2022`.
+
+### Setup Instructions
+
+**Dependency:** `containerd` v1.7.12+
```

Comment:
I think following https://github.com/moby/buildkit/pull/4652, we can include `containerd.exe` in windows binaries bundle so we don't need to update this docs frequently. I'll open a follow-up to include `containerd.exe` in the packaging if that sounds good. We could then say that user can either use containerd inside buildkit bundle or official 1.7.12+.